src/webextension/firstrun/components/app.css

```diff
@@ -5,7 +5,10 @@
 body {
   padding: 0;
   margin: 0;
+  color: #0c0c0d;
   background-color: #f9f9fa;
+  font: caption;
+  -moz-user-select: none;
```

Comment:
Mentioned at https://github.com/mozilla-lockbox/lockbox-extension/issues/311 this prevents users from selecting text such as the web address/domain on the entry view.